src/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ add_compile_definitions(QT_DISABLE_DEPRECATED_BEFORE=0x050F00)
 
 # registers the native messageing host
 option(CHATTERINO_DEBUG_NATIVE_MESSAGES "Debug native messages" OFF)
+option(DISABLE_AUTOMATIC_UPDATES "Disable automatic updates" OFF)
```

Comment:
I'd prefer if this was named `CHATTERINO_DISABLE_UPDATER` and defined in the root CMakeLists.txt, underneath the `CHATTERINO_PLUGIN` option